broker/utils.go

```diff
@@ -25,10 +25,6 @@ func findDeployment(c compose.Client, name string) (*composeapi.Deployment, erro
 	return nil, fmt.Errorf("deployment: not found")
 }
 
-func JDBCURI(scheme, hostname, port, dbname, username, password string) string {
```

Comment:
This is not a MongoDB broker but Compose broker. We want tot use it for other databases. What about JDBC support for elasticsearch, redis and others?